pkgs/games/steam/steam.nix

```diff
@@ -27,6 +27,12 @@ in stdenv.mkDerivation {
       chmod +x $out/bin/steamdeps
     ''}
 
+    # install udev rules
+    mkdir -p $out/etc/udev/rules.d/
+    cp ./subprojects/steam-devices/*.rules $out/etc/udev/rules.d/
+    substituteInPlace $out/etc/udev/rules.d/60-steam-input.rules \
+      --replace "/bin/sh" "${bash}/bin/bash"
```

Comment:
Do you think it is safe to replace sh with bash?

Reply:
yes, sh is a subset of bash

Reply:
and on NixOS:
```
$ realpath /bin/sh
/nix/store/j37555sj2w3xsql3f8qrwbaim7pv67hg-bash-interactive-4.4-p23/bin/bash
```